egs/wenetspeech/s5/local/score.sh

```diff
@@ -0,0 +1,201 @@
+#!/bin/bash
```

Comment:
@LvHang, as an option, if that's the only difference, you may call a python file "local/character_tokenizer" without ".py"; the system uses shebang to find the interpreter. I routinely do that all the time for little CLI tools I write for myself¹. Emacs is smart enough to recognize the shebang and select a major mode; most editors allow specifying a hint comment in the file as to what language support to select. Or, alternatively, symlink `local/character_tokenizer -> local/character_tokenizer.py` if you have to keep the extension `.py`. 

Totally up to you; your benefit is it will shift the maintenance burden from you. Anything in `local/` is yours to maintain, but if you symlink to e.g. `../steps/....` it maintains itself from your perspective (i.e., we do that). 

---
¹ E.g. this. Note I do not have to instruct Emacs to use python-mode, only set a non-standard indent, to offend the PEP-wielding pythonistas.
```py
~$ head $(type -p j2y)
#!/usr/bin/env python3
# -*- python-indent-offset: 2; -*-
# Requires apt install python3-yaml

import argparse as ap, json, signal as sig, sys, yaml

# Don't puke stack trace on ctrl+C
sig.signal(sig.SIGINT, sig.SIG_DFL)
  .  .  .  .
```